### DISCUSSION TITLE
Tiled VAE Decode

### DISCUSSION THREAD

**Discussion by fabian2665** [ssitu/ComfyUI_UltimateSDUpscale#40] [Q&A] [answered]
I use ComfyUI on a low-end machine and my vram isn't enough for the normal VAE Decode. Tiled VAE Decode works well though. Would it be possible to have a setting to always use Tiled Decode? As it is right now, it always tries normal Decode and reverts to tiled Decode as it runs out of memory. I might be wrong but i feel like it adds quite a bit of processing time.

**Comment by ssitu** [comment, marked as answer]
I’ll add a toggle at the end for this if that sounds good to you.

**Reply by fabian2665** [reply]
That would be perfect! Thank you so much for your effort. :)

**Reply by ssitu** [reply]
It is in! I'd like to know if it does the job.

**Reply by fabian2665** [reply]
I tested it a couple of times and it's exactly how I imagined it. Thank you so much for implementing it that quickly. I really appreciate your work.
